### DISCUSSION TITLE
Is there an option like --assume-yes for fastlane deploy?

### DISCUSSION THREAD

**Discussion by nohli** [fastlane/fastlane#17733] [Q&A] [answered]
Using fastlane deploy with a launchd script for nightly builds.

Since I'm using the swift beta for two repositories, it asks to update fastlane like once a week...and if you don't manually say yes, the script will never finish.

In apt-get there is an option `--assume-yes`. Is there something alike in fastlane deploy?

**Comment by rogerluan** [comment, marked as answer]
Hey @nohli 👋 

That feature exists and it's an environment variable called `FASTLANE_SKIP_UPDATE_CHECK`. You can read more about it here: https://docs.fastlane.tools/advanced/fastlane/#skip-update-check-when-launching-fastlane

Hope it helps!

**Reply by nohli** [reply]
Thanks! It does the job to finish the script.

A perfect solution would be to assume yes if fastlane wants to update.
